network-api/networkapi/templates/fragments/buyersguide/external_link_icon.html

```diff
@@ -0,0 +1,2 @@
+{% load static i18n %}
```

Comment:
Great idea adding this to its own template!

Reply:
Thanks @danielfmiranda. Yea, I think small very reusable templates make it easier to keep out components consistent across the code base. 

Of course this one is static, but the approach also works for template with parameters is we keep the parameters as flat as possible and don't rely on the template "knowing" about where and how it is used. E.g. for flat parameters we could use`{{ title }}` instead of `{{ page.title }}` in the templates, and then pass the right value for `title` with something like `{% include "fragments/my_template.html" with title=page.title %}`.